substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/BundleSupport.java

```diff
@@ -586,6 +706,27 @@ private Path writeBundle() {
             nativeImage.deleteAllFiles(metaInfDir);
         }
 
+        Path bundleLauncherFile = Paths.get("/").resolve(BundleLauncher.class.getName().replace(".", "/") + ".class");
+        try (FileSystem fs = FileSystems.newFileSystem(BundleSupport.class.getResource(bundleLauncherFile.toString()).toURI(), new HashMap<>());
```

Comment:
Since you already copy together some .class files, why not also copy the regular `JSONParser`? Just because it is using `EconomicMap` (for which there is no good reason other than a few people believing it is so much better than `HashMap`)?